djstripe/stripe_objects.py

```diff
@@ -407,10 +411,13 @@ def sync_from_stripe_data(cls, data):
         return instance
 
     def __str__(self):
+        """Return string repr of StripeObject."""
```

Comment:
Return a string representation of a Stripe object using the ``str_parts()`` method.